python/GafferUI/ErrorDialogue.py

```diff
@@ -43,44 +43,102 @@
 
 class ErrorDialogue( GafferUI.Dialogue ) :
 
-	def __init__( self, title, message, details=None, **kw ) :
+	def __init__( self, title, message = None, details = None, messages = None, closeLabel = "Close", **kw ) :
```

Comment:
Yeah, I wasn't particularly happy about that - it was actually the reason I haven't done this until now. But since the alternative was to copy the custom error management from FileMenu into even more places, I decided the additional argument was the lesser of the evils. I've pushed some documentation in a87d927.

I do hope that the standard use pattern for the ErrorDialogue will always be `with ErrorDialogue.ErrorHandler()` and then these details just drop away.